docs/sidebars.yaml

```diff
@@ -20,7 +20,7 @@
         - page: tutorials/quickstart/mint-and-burn-nftokens.md
         - page: tutorials/quickstart/transfer-nftokens.md
         - page: tutorials/quickstart/broker-sale.md
-        - page: tutorials/quickstart/authorize-minter.md
+        - page: tutorials/quickstart/authorize-minter.md     
```

Comment:
Nit: try not to introduce extraneous whitespace like this. There might be an editor setting you could enable to automatically remove it (like what Atom does).